release_notes.md

```diff
@@ -6,3 +6,22 @@
     - The entire feeds database is labelled with the relevant threat types, and associated LockHeed Martin Kill Chain Phases, that enables user with a lot of contextual information to understand the nature of threat.
 
 - With added support for the FortiGuard Feeds, the **Preview** label has been removed from the Solution Pack.
+
+# Known Issues
+
+- Data ingestion with MalSilo v2.0.0 and older may fail after an upgrade to v7.3.0.
+
+As a workaround you may perform the following steps:
+
+1. Login to FortiSOAR.
+2. Click **Automation** > **Playbooks**.
+3. Search and open the MalSilo playbook collection. The collection's name is in the format `Sample - MalSilo - <version>`.
```

Comment:
The change has to be done in the configured Data Ingestion playbooks and not the Sample